helper/validation.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+function image_check($pic){
+    
+    $uploadOk = 1;
```

Comment:
Keep variable names consistent
variable $uploadOk should be like $upload_ok or $is_upload_allowed or $is_image_valid